feedback-module/src/lib/hooks/useInputs.js

```diff
@@ -60,8 +61,14 @@ export default function useInputs(firstCheckRef) {
         invalidPhone(question.answer, question.required)
       ) {
         (validated = false), (question.error = true);
+      } else if (
+        question.type === "textarea" &&
+        question.answer.length > TEXTAREA_MAX_CHAR
+      ) {
+        (validated = false), (question.charError = true);
```

Comment:
Is there a reason we need this to be charError instead of just error? Also, I noticed that there are a lot of repeated checks going on between this `inputsValidated` function and the `inputError` function in `TextboxList` (for example, we check `question.type` here and `input.type` in Modules.js. I feel like it would make things simpler if instead of `question.error` being a boolean, we set `question.error` to be the actual error message key or undefined if there is none. Then, instead of doing all these checks again in the `inputError` function and separating out `error` and `charError`, we could just do a simple
```
if (inputQuestions && inputQuestions[index] && inputQuestions[index].error) {
const key = inputQuestions[index].error;
return (
      <ErrorAlert
        key={key}
        errorText={t(`errorMessages.${key}`)}
        id={id}
        className="margin-bottom-2"
      />
    );
}

Reply:
Also, probably not a big deal, but the `validated` variable seems a little unnecessary to me and seems kind of redundant to have to set it in each if/else if body. I think it might be cleaner if we just return
```
return !inputQuestions.some(question => question.error);
```
at the very end